### DISCUSSION TITLE
Cannot add a list item to the query parameters in SwaggerUI if the parameter is a list of enums

### DISCUSSION THREAD

**Discussion by bogdan-coman-uv** [fastapi/fastapi#11025] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, Query
from typing import Annotated
from enum import Enum

class TestEnum(str, Enum):
    A = "a"
    B = "b"
    C = "c"

app = FastAPI()

@app.get("/")
async def read_items(q: Annotated[list[TestEnum], Query()]=None):
    results = {"items": [{"item_id": "Foo"}, {"item_id": "Bar"}]}
    if q:
        results.update({"q": q})
    return results
```

### Description

I open the SwaggerUI docs and expect to be able to add items to the q parameter, but the "Add list item" button is missing. 
![image](https://github.com/tiangolo/fastapi/assets/92912507/7e0242f2-c734-48d1-8db0-47baf6a6bafd)

However, if I define q as list[str] I can see the button:
```
@app.get("/")
async def read_items(q: Annotated[list[str], Query()]=None):
    results = {"items": [{"item_id": "Foo"}, {"item_id": "Bar"}]}
    if q:
        results.update({"q": q})
    return results
```
![image](https://github.com/tiangolo/fastapi/assets/92912507/795bc922-63a4-4c3c-b143-a7f57337ba58)

### Operating System

Linux

### Operating System Details

Ubuntu 22.04.3 LTS on WSL 

### FastAPI Version

0.109.0

### Pydantic Version

2.5.3

### Python Version

3.11.7

**Comment by n8sty** [comment, marked as answer]
You're right that you don't get the "Add string item" or similar button but if you choose the options you want while holding down ctrl or whatever the equivalent is on your OS you should be able to select multiple items. If you submit the request you can see that multiple items are included.

**Reply by bogdan-coman-uv** [reply]
Yep, it works. Thank you
